src/include/planner/planner.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "binder/bound_statement.h"
+#include "binder/statement/select_statement.h"
```

Comment:
I feel like we can remove a few of these.. but the planner is hopefully not included in very many places, so no strong opinion.

Reply:
Yes, that's why I don't care much about forward declaration for this file 🤣 But I'll still try forwarding decl more.